### PR TITLE
JDK-8304867: Explicitly disable dtrace for ppc builds

### PR DESCRIPTION
**Pull request by MBaesken**
Currently the support for the JVM feature 'dtrace' is not implemented in the HS codebase on ppc, so we should disable it in the jvm-features make detection as well. Otherwise we could run into building with dtrace feature support if e.g. the 'dtrace' tool and related header(s) are available on the build machine.

Probably the same is true for the s390x platform, see
https://github.com/openjdk/jdk/blob/0deb648985b018653ccdaf193dc13b3cf21c088a/src/hotspot/cpu/s390/c1_LIRAssembler_s390.cpp#L226
but I only looked a bit more into ppc.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8304867](https://bugs.openjdk.org/browse/JDK-8304867): Explicitly disable dtrace for ppc builds

### Reviewers
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**)
 * [Aleksey Shipilev](https://openjdk.org/census#shade) (@shipilev - **Reviewer**)
 * [Lutz Schmidt](https://openjdk.org/census#lucy) (@RealLucy - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13173/head:pull/13173` \
`$ git checkout pull/13173`

Update a local copy of the PR: \
`$ git checkout pull/13173` \
`$ git pull https://git.openjdk.org/jdk.git pull/13173/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13173`

View PR using the GUI difftool: \
`$ git pr show -t 13173`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13173.diff">https://git.openjdk.org/jdk/pull/13173.diff</a>

</details>

### CODE REVIEW

File: make/autoconf/jvm-features.m4 [hunk 1]
```diff
@@ -248,7 +248,10 @@ AC_DEFUN_ONCE([JVM_FEATURES_CHECK_DTRACE],
 [
   JVM_FEATURES_CHECK_AVAILABILITY(dtrace, [
     AC_MSG_CHECKING([for dtrace tool])
-    if test "x$DTRACE" != "x" && test -x "$DTRACE"; then
+    if test "x$OPENJDK_TARGET_CPU_ARCH" = "xppc"; then
+      AC_MSG_RESULT([no])
```
Comment (shipilev) [inline make/autoconf/jvm-features.m4:252]: Look at how the block above replies the OS-CPU, to highlight that we don't have this support due to arch. Should do the same here.
https://github.com/openjdk/jdk/pull/13173/files#diff-a09b08bcd422d0a8fb32a95ccf85051ac1e69bef2bd420d579f74d8efa286d2fR236

```
AC_MSG_RESULT([no, $OPENJDK_TARGET_OS-$OPENJDK_TARGET_CPU])
```

**Comment by MBaesken** [comment]
Hi Aleksey, I adjusted the output.

**Review by shipilev — APPROVED** [review]

**Review by RealLucy — APPROVED** [review]
Looks good to me.
Will you take care of s390x in a separate PR? Or will @offamitkumar handle it?

**Comment by offamitkumar** [comment]
I can take it, if @MBaesken agrees :-)

**Comment by MBaesken** [comment]
> I can take it, if @MBaesken agrees :-)

I agree !

**Comment by MBaesken** [comment]
Thanks for the reviews !

/integrate

**Pull request closed without merging**